tests/e2e/testcases/network/clean/suite.go

```diff
@@ -0,0 +1,27 @@
+// Copyright (C) 2025, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package network
+
+import (
+	"github.com/ava-labs/avalanche-cli/tests/e2e/commands"
+	ginkgo "github.com/onsi/ginkgo/v2"
+	"github.com/onsi/gomega"
+)
+
+var _ = ginkgo.Describe("[Local Network] Clean", ginkgo.Ordered, func() {
+	ginkgo.It("can clean a started network", func() {
+		out := commands.StartNetwork()
+		gomega.Expect(out).Should(gomega.ContainSubstring("Network ready to use"))
+
+		out, err := commands.CleanNetwork()
+		gomega.Expect(err).Should(gomega.BeNil())
+		gomega.Expect(out).Should(gomega.ContainSubstring("Process terminated"))
+	})
+
```

Comment:
can we add test for this flag too
````
cmd.Flags().BoolVar(
		&hard,
		"hard",
		false,
		"Also clean downloaded avalanchego and plugin binaries",
	)

Reply:
added in

Reply:
the comments here say `plugin bins` are also cleaned up, but per https://github.com/ava-labs/avalanche-cli/blob/main/cmd/networkcmd/clean.go#L65 we only clean ava-go bins?

Reply:
can we follow up with @felipemadero to see if this is a bug?

Reply:
most probably this flag should be deprecated

Reply:
the above PR closed - addressed